pkg/filenotify/poller_test.go

```diff
@@ -68,6 +68,14 @@ func TestPollerEvent(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	// FIXME this should really be `0600`, but fixing that makes the test fail
+	// SA9002: file mode '600' evaluates to 01130; did you mean '0600'? (staticcheck)
+	// changing to 0600 causes;
+	//
+	// === Failed
+	// === FAIL: pkg/filenotify TestPollerEvent (0.80s)
+	//     poller_test.go:75: timeout waiting for event CHMOD
```

Comment:
I guess it fails because the file already had `0600` perm bits, so chmod has no effect and the event is not generated. Please use say `0640` (aka `-rw-r------`)

Reply:
Right, so that's what I thought as well, but then looked at the line above, where the file is created with `0644`. https://github.com/moby/moby/blob/8e610b2b55bfd1bfa9436ab110d311f5e8a74dcb/pkg/filenotify/poller_test.go#L64-L73

Perhaps it's because due to umask? haven't checked that one

Reply:
Yup, looks to be umask, so the file is _created_ with `0600` (added an assert)

```
=== Failed
=== FAIL: pkg/filenotify TestPollerEvent (0.00s)
    poller_test.go:67: expected file /tmp/test-poller830233858 to have mode 0644, but got 0600
```